packages/calcite-components/src/components/action-bar/action-bar.tsx

```diff
@@ -292,6 +295,13 @@ export class ActionBar extends LitElement {
 
   private handleDefaultSlotChange(): void {
     this.updateGroups();
+    if (this.layout === "horizontal-fill") {
+      const actions = Array.from(this.el.querySelectorAll("calcite-action"));
```

Comment:
can you use the `queryActions()` util here to get the actions? So it doesn't set the fullWidth on actions within action-menu?

Reply:
There may need to be more work done here so that the alignment and fullWidth are removed when the action's slot is changed to be in the action-menu.